templates/algob-react-boilerplate/config-overrides.js

```diff
@@ -0,0 +1,8 @@
+const ModuleScopePlugin = require('react-dev-utils/ModuleScopePlugin');
+const rewireYAML = require('react-app-rewire-yaml');
+
+module.exports = function override(config, env) {
+	config.resolve.plugins = config.resolve.plugins.filter(plugin => !(plugin instanceof ModuleScopePlugin));
+	config = rewireYAML(config, env);
```

Comment:
Is this to load `ASA` definition?  What other stuff do we have in YAML?

Reply:
Yes, basically to load the deployed information from `cp` in `/artifacts` (by importing directly in react app). All deployed info is stored in `yaml` files (ASA definition, SSC info, contract address, delegated signture info).